src/components/Modal/Modal.tsx

```diff
@@ -29,7 +29,7 @@ export interface ModalProps {
 	children: ReactNode;
 	isOpen: boolean;
 	title?: string;
-	size?: 'small' | 'medium' | 'fullscreen' | 'auto';
+	size?: 'small' | 'medium' | 'large' | 'fullscreen' | 'auto';
```

Comment:
Update tests. (+ er is ook nog de nieuwe `.c-modal--fullwidth` & `.c-modal--scrollable`